en/02_Developer_Guides/09_Security/05_Secure_Coding.md

```diff
@@ -578,47 +578,58 @@ salt values generated with the strongest entropy generators available on the pla
 (see [RandomGenerator](api:SilverStripe\Security\RandomGenerator)). This prevents brute force attacks with
 [Rainbow tables](https://en.wikipedia.org/wiki/Rainbow_table).
 
-Strong passwords are a crucial part of any system security. So in addition to storing the password in a secure fashion,
-you can also enforce specific password policies by configuring a
-[PasswordValidator](api:SilverStripe\Security\PasswordValidator). This can be done through a `_config.php` file
-at runtime, or via YAML configuration.
+Strong passwords are a crucial part of any system security.
 
-The default password validation rules are configured in the framework's `passwords.yml`
-file. You will need to ensure that your config file is processed after it.
+The default password validator uses the [`PasswordStrength` constraint](https://symfony.com/doc/current/reference/constraints/PasswordStrength.html) in `symfony/validator`, which determines a password's strength based on its level of entropy.
+
+You can change the required strength of valid passwords by setting the [`EntropyPasswordValidator.password_strength`](api:SilverStripe\Security\Validation\EntropyPasswordValidator->password_strength) configuration property to one of the valid [minScore values](https://symfony.com/doc/current/reference/constraints/PasswordStrength.html#minscore):
+
+```yml
+SilverStripe\Security\Validation\EntropyPasswordValidator:
+  password_strength: 4
+```
+
+You can also enforce that passwords are not repeated by setting the [`EntropyPasswordValidator.historic_count`](api:SilverStripe\Security\Validation\EntropyPasswordValidator->historic_count) configuration property:
+
+```yml
+SilverStripe\Security\Validation\EntropyPasswordValidator:
+  historic_count: 6
```

Comment:
```suggestion
You can also enforce that passwords are not repeated by setting the [`EntropyPasswordValidator.historic_count`](api:SilverStripe\Security\Validation\PasswordValidator->historic_count) configuration property:

```yml
SilverStripe\Security\Validation\PasswordValidator:
  historic_count: 6
```

Configuration is defined on PasswordValidator, and also works if it password validator is changed to RulesPasswordValidator or a custom password validator

I assume this works if we set it on the abstract class?

Reply:
It should work if set on the abstract class, yeah. Though IMO it makes more sense to set it on whichever validator is in use - i.e. you set the validation you want for the validator you want to use.

Given the sections are currently split up to discuss the individual validators, I'd like to keep the config being set _for_ those validators - I think it'd read weirdly in a section about one validator you're setting config on its parent class.

All that said, I don't feel strongly so if you still want it changed I can change it.

Reply:
I'd say change it to the abstract class, as if people change the validator in the future they'll retain the historic password count config

Reply:
Done